test/tests/providers/capz/cert-manager.rules.test.yml

```diff
@@ -0,0 +1,48 @@
+---
+rule_files:
+  - cert-manager.rules.yml
```

Comment:
would it make sense to push the `cert-manager` tests separately ?

Reply:
i've done this in a seperate commit:
https://github.com/giantswarm/prometheus-rules/pull/596/commits

but i'm also fine to create a dedicated PR for that

Reply:
separate commits sounds good, i should have checked